Sources/SwiftMemcache/MemcachedConnection.swift

```diff
@@ -184,10 +192,20 @@ public actor MemcachedConnection {
             if var unwrappedResponse = response {
                 return Value.readFromBuffer(&unwrappedResponse)
             } else {
-                throw MemcachedConnectionError.unexpectedNilResponse
+                throw MemcachedError(
+                    code: .unexpectedNilResponse,
```

Comment:
Clarifying my other comment here. Instead of throwing `unexpectedNilResponse` let's name this error `protocolError` instead.